ui/src/components/App.spec.js

```diff
@@ -19,11 +22,16 @@ const setup = (propOverrides) => {
   };
 };
 
+/**
+ * @todo get these tests working
+ */
 describe('components', () => {
   describe('App', () => {
     it('should render', () => {
-      const { output } = setup();
-      expect(output.type).toBe(Router);
+      const { output } = setup({ isLoggedIn: true, isLoading: true });
+
+      const app = output.props.children;
```

Comment:
```suggestion

```
This line should no longer be necessary, as you do not need to drill down into the `output` object, as the `output` object should just render a `Router` object as per the render method, which you can test for as was done previously.